readthedocs-theme/templates/includes/footer.html

```diff
@@ -0,0 +1,170 @@
+<footer class="ui basic inverted segment very padded" id="footer">
+  <div class="ui container">
+
+      <div class="ui three column stackable doubling grid information very padded">
+
+        <div class="column">
+          <h4 class="ui sub header">Stay Updated</h4>
+
+          <div class="ui list">
+            <a href="https://blog.readthedocs.com/" class="item">Blog</a>
+            <div class="item">
+              <a href="https://github.com/rtfd/readthedocs.org" rel="noopener noreferrer"><i class="icon large light github"></i></a>
+              <a href="https://twitter.com/readthedocs" rel="noopener noreferrer"><i class="icon large light twitter"></i></a>
+            </div>
+          </div>
+
+          <div class="ui form">
+            <form id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+              <p>Sign up for our newsletter to get our latest blog updates delivered to your inbox weekly.</p>
```

Comment:
Might be best to drop the timeframe, we're not yet at a weekly cadence for newletters

```suggestion
              <p>Sign up for our newsletter to get our latest blog updates delivered to your inbox.</p>
```